.github/dependabot.yml

```diff
@@ -20,4 +20,4 @@ updates:
     # future if we reorganise teams
     labels:
       - "dependencies"
-      - "Team-Magma"
```

Comment:
The comment above is now outdated

Reply:
True, but the effort to remove it is too high tbh.